nbs/models.hint.ipynb

```diff
@@ -366,12 +366,15 @@
     "        \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #66.            assert np.allclose(children_sum, parent_value)
</pre><p>I think we should use <code>np.testing.assert_allclose</code>, since that has a lower tolerance and we're looking to get as close as possible here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/964/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Ok, changed it, although it might give issues at some point given we're using float32 most of the time.